.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
```

Comment:
Can you add `yarn.lock` here? So that I can use yarn to run tests on development.

Reply:
Done